src/coreclr/vm/ceemain.cpp

```diff
@@ -871,6 +871,9 @@ void EEStartupHelper()
         // requires write barriers to have been set up on x86, which happens as part
         // of InitJITHelpers1.
         hr = g_pGCHeap->Initialize();
+
+        g_pConfig->SetGCLOHThreshold((DWORD)GCHeapUtilities::GetGCHeap()->GetLOHThreshold());
```

Comment:
It looks odd to cache the value on `g_pConfig`. It would look better to cache it in a static variable in gcheaputilities.* or gchelpers.*.

Reply:
This change should be just about LOHThreshold. My point is that LOHThreshold is not read from the config by EEConfig after this change, so there is no point in keeping the cached value for it on EEConfig. After this change, LOHThreshold is yet another value computed by the GC that the VM decided to cache.

The other flags in your list are read from the config by EEConfig so I do not see a problem with keeping those on EEConfig.

BTW: None of the few places where LOHThreshold is used on the VM side look perf sensitive. I think it would ok to just call the GC API in these places, without caching the value ahead of time. It is fine to cache it if you like it better - I do not have a problem with it.